franklin/github/views.py

```diff
@@ -31,7 +32,7 @@ def repository_list(request):
             # TODO - return error from github
             request.user.details.update_repos_for_user(github_repos)
         sites = request.user.details.sites.filter(is_active=True).all()
-        site_serializer = SiteSerializer(sites, many=True)
+        site_serializer = FlatSiteSerializer(sites, many=True)
```

Comment:
return less data so this "list all repos" endpoint doesn't return too much data or accidentally call github 50 times.